src/components/transactions.rs

```diff
@@ -50,12 +51,16 @@ pub fn mine_block(
 }
 
 /// Mine the given number of empty blocks and return the block height and coinbase txid of the first block
-pub fn mine_empty_blocks(num_blocks: u32, rpc_client: &mut dyn RpcClient) -> (u32, TxId) {
+pub fn mine_empty_blocks(
+    num_blocks: u32,
+    rpc_client: &mut dyn RpcClient,
+    activate: bool,
+) -> (u32, TxId) {
     if num_blocks == 0 {
         panic!("num_blocks must be greater than 0")
     }
 
-    let (block_height, coinbase_txid) = mine_block(rpc_client, BranchId::Nu5, vec![], false);
+    let (block_height, coinbase_txid) = mine_block(rpc_client, BranchId::Nu5, vec![], activate);
```

Comment:
Actually, in new versions of tx we don't need it anymore except from the very last step where this is ignored.  I'll hide it deeper to avoid confusion